ibis/backends/tests/test_array.py

```diff
@@ -5,9 +5,8 @@
 import ibis.expr.types as ir
 
 
-@pytest.mark.xfail_unsupported
-@pytest.mark.skip_missing_feature(
-    ['supports_arrays', 'supports_arrays_outside_of_select']
+@pytest.mark.backends_notimpl(
+    ["sqlite", "mysql", "impala", "postgres", "clickhouse", "datafusion"]
 )
 def test_array_column(backend, alltypes, df):
```

Comment:
If there's a way to mark `sqlite`, and `mysql` as "never" for all array operations that seems reasonable. Neither of those backends is likely to ever implement support for array types.